RecoEgamma/PhotonIdentification/plugins/PhotonIDValueMapProducer.cc

```diff
@@ -67,7 +67,7 @@ class PhotonIDValueMapProducer : public edm::stream::EDProducer<> {
   float computeWorstPFChargedIsolation(const T& photon,
 				       const U& pfCandidates,
 				       const edm::Handle<reco::VertexCollection> vertices,
-				       bool isAOD,
+				       bool isAOD, const reco::Vertex pv,
```

Comment:
```const reco::Vertex& pv```
to avoid copies

Reply:
Changes implemented here:
https://github.com/lsoffi/cmssw/blob/CMSSW_9_4_0_pre3_TnP/RecoEgamma/PhotonIdentification/plugins/PhotonIDValueMapProducer.cc